client/src/pages/messaging/main/index.js

```diff
@@ -78,22 +80,72 @@ const Chat = () => {
         ))}
       </div>
       <br />
-      <div>
-        {messages.map(({ body, sender }, i) => (
-          <div key={i}>
-            <b>{sender === userId ? 'You: ' : 'They: '}</b>
-            {body}
+      <div
```

Comment:
Looks like you forgot to replace with css classes

Reply:
No. The changes were not getting reflected in the ui